<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registrationDialog</class>
 <widget class="QDialog" name="registrationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1263</width>
    <height>864</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="3">
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="btnSelectSource">
       <property name="text">
        <string>Source</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Source</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="btnSelectInput">
       <property name="text">
        <string>Image to Register</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="btnSaveRegistration">
       <property name="text">
        <string>Save Registration</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Registration Technique</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QComboBox" name="technique"/>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Display Method</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="displayMethod"/>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Boarder Method</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QComboBox" name="boarderMethod"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Input</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" rowspan="3" colspan="2">
      <widget class="imageEdit" name="registered">
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>240</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Registered</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="imageEdit" name="source">
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>240</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="imageEdit" name="input">
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>240</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>imageEdit</class>
   <extends>QLabel</extends>
   <header>imageedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
